doc/nrf/releases/release-notes-latest.rst

```diff
@@ -141,6 +141,52 @@ Matter (Project CHIP)
 
   * Renamed occurrences of Project CHIP to Matter.
 
+Bluetooth Mesh
+--------------
+
+* Added:
+
+  * Support for vendor specific mesh model :ref:`bt_mesh_silvair_enocean_srv_readme`.
+  * API function to manually store pending RPL entries in the persistent storage without waiting for the timeout.
+
+* Updated:
+
+  * Fixed an issue by :ref:`bt_mesh_light_xyl_srv_readme` publishes its state values that was loaded from flash after power up.
+  * Replaced the linked list of scene entries in the model contexts, with a lookup in ROM-allocated scene entries for each model in the composition data when storing and restoring.
+  * Updated so the transition pointer can be NULL, if no transition time parameters are provided in APIs.
```

Comment:
This will not be correct to write "to be", because it now can be NULL. Before it could not be NULL. Now it can be NULL, but it can be something else too.